src/uct/ib/rc/base/rc_iface.c

```diff
@@ -591,14 +593,15 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_iface_ops_t *tl_ops,
 #endif
     max_ib_msg_size             = uct_ib_iface_port_attr(&self->super)->max_msg_sz;
 
-    self->config.conn_ece = 0;
     if (dev->flags & UCT_IB_DEVICE_FLAG_ECE) {
-        if (config->conn_ece == UCS_ULUNITS_AUTO) {
-            self->config.conn_ece = 0xffffffff;
+        if (config->ece == UCS_ULUNITS_AUTO) {
+            self->config.ece = 0xffffffffU;
```

Comment:
add to ib_device.h:
`#define UCT_IB_DEVICE_ECE_MIN 0x0`
`#define UCT_IB_DEVICE_ECE_MAX 0xffffffffU`

and use them instead of 0xff..f and 0 for ece

Reply:
@yosefe For ```input ece with 0```, could we use ```#define UCT_IB_DEVICE_ECE_DEFAULT 0x0``` instead of ```UCT_IB_DEVICE_ECE_MIN```? ```0``` means it keep the default ECE.

Reply:
Done